sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -10,6 +10,13 @@
 - Removed `[remote]` extra. This is no longer needed when tracking results in Azure AI Studio.
 - Fixed `AttributeError: 'NoneType' object has no attribute 'get'` while running simulator with 1000+ results
 - Fixed the non adversarial simulator to run in task-free mode
+- Content safety evaluators (violence, self harm, sexual, hate/unfairness) return the maximum result as the
+  main score when aggregating multiple per-turn evaluations from a conversation into a single overall
+  evaluation score. Other conversation-capable evaluators still default to a mean.
+- Evaluator aggregation for producing a main score multi-turn conversations is now configurable. Can be configured
+  quickly by providing an Enum value (`~azure.ai.evaluation.AggregationType`) via
+  `eval.set_conversation_aggregation_type` or the exact aggregation function can be assigned directly via
```

Comment:
shouldn't it be `_set_conversation_aggregation_type`?

Reply:
That said, this is no longer relevant since I'm removing this line from the CL. I want to save announcing user-configurability for a later time.